node/core/dispute-coordinator/src/scraping/candidates.rs

```diff
@@ -7,7 +7,15 @@ use std::collections::{BTreeMap, HashMap, HashSet};
 /// Each `remove` decreases the reference count for the corresponding `CandidateHash`.
 /// If the reference count reaches 0 - the value is removed.
 struct RefCountedCandidates {
-	candidates: HashMap<CandidateHash, usize>,
+	candidates: HashMap<CandidateHash, CandidateInfo>,
+}
+
+// If a dispute is concluded against this candidate, then the ChainSelection
+// subsystem needs block number and block hash to mark the relay parent as reverted.
+pub struct CandidateInfo {
+	count: usize,
```

Comment:
Personal opinion: I'd name this `reference_count` or `ref_count`. `count` is a bit confusing.

Reply:
Agreed!